package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "azuredatastudio",
   "version": "1.30.0",
-  "distro": "ab16639070613df1c9772fe028786f44ff9096f3",
+  "distro": "da9346f4dbf9e6069dd59f3d5b2ee4dcc1a19b2a",
```

Comment:
Is this part of the test changes?

Reply:
Yes, there was a merge conflict in the distro repo I had to fix